package/MDAnalysis/core/universe.py

```diff
@@ -738,7 +738,7 @@ def _gen_anchor_hash(self):
                 return self._anchor_uuid
             except AttributeError:
                 # store this so we can later recall it if needed
-                self._anchor_uuid = uuid.uuid4()
+                self._anchor_uuid = str(uuid.uuid4())
```

Comment:
uuid's didn't seem to survive the serialisation well, but if I just take a string it's all good (and still uuidesque)